### ISSUE TITLE
Can't add or update product with declinations that lead to an 403 Error

### ISSUE THREAD

**Issue by jnvgi** [PrestaShop/PrestaShop#21846]
Hello,

I can't update or add products with declinations. My PS version was a bit of an old one, so i updated to 1.7.6.8 but issue is still there.

When i activate the Debug mode, i can see there is a 403 error, but can't get anymore information about it. What's even worse is that once i insist, i get a 403 Forbidden on all the website from the selected browser. So I switched to another browser and got 403 a few minutes later. 
Basically now i'm totally stucked with a website I can't reach from my computer (still reachable from other PC).

At really first, i just wented to clear the cache and set up a CDN. I didn't install any plugins.

Seems like this bug with the products with declinations that can't be updated is a common thing but I couldn't find a way to solve it.

* PrestaShop version: 1.7.6.8
* PHP version: 7.3
* Site URL : www.phone-united.com

**Comment by hibatallahAouadni** [comment]
Hello @jnvgi ,

We use GitHub issues only to discuss about bugs and new features in the PrestaShop project. If you have questions about using PrestaShop or third-party modules, or if you need help with your shop, please consider one of our [support plans](https://www.prestashop.com/support).

Alternatively, you can also ask for help in the [community forums](https://www.prestashop.com/forums/) or in the [public Slack channel](https://github.com/PrestaShop/open-source/tree/master/slack).

Thanks!

**Comment by jnvgi** [comment]
Well I understand your point but it clearly seems that the fact that we can't update a product with declination is a recurrent problem in Prestashop...

**Comment by hibatallahAouadni** [comment]
Hi @jnvgi 

I didn't manage to reproduce your issue with PS1.7.4.2, PS1.7.5.2, PS1.7.6.8 and PS1.7.7.0. It means that in all versions we can easily update product with combinations.

By the way, you need to downgrade your PHP version to 7.2 cause PS1.7.6.8 is not compatible with PHP7.3 (see the system requirements [link](https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/)).

Thanks!
